pkgs/by-name/sh/shh/package.nix

```
@@ -42,8 +82,23 @@ rustPlatform.buildRustPackage rec {
     iproute2
   ];
 
+  # todo elvish
+  postInstall = ''
```

```suggestion
  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
```

---

I would honestly prefer if the package cross compiled with an error and not silently drop the docs

---

fixed cross compilation in latest commit

---

> not silently drop the docs

You can use this

https://github.com/NixOS/nixpkgs/blob/863da33c927ad15a242d862e02d057d33c41bfb1/pkgs/by-name/uv/uv/package.nix#L55-L65

> I would honestly prefer if the package cross compiled with an error

Known errors should be thrown at eval time.

---

Honestly, I kinda bypassed this issue by purpouse compiling the binary for native when a cross vompike or build without doc features is detected